todo_app/templates/index.html

```diff
@@ -12,7 +12,33 @@ <h1 class="display-4">To-Do App</h1>
       <h2>Items</h2>
       <ul class="list-group mb-4">
         <!-- List the todo items here -->
+        <table>
+          <tr>
+              {%for col in data[0].keys()%}
+                <th>{{ col }}</th>
+              {%endfor%}
+          </tr>
+          {%for i in range(0, len)%}
+            <tr>
+              {%for col in data[i].values()%}
+                <td>{{ col }}</td>
+              {%endfor%}
+            </tr>
+          {%endfor%}
+        </table>
       </ul>
     </div>
   </div>
+
+  <div class="row">
+    <div class="col-md-12">
+      <h2>Add New Item</h2>
+      <form action="{{ url_for('add_new_item') }}" method="post">
+        Status: <input type="text" name="item_status" style=width:100px value="Not Started" readonly>
+        Title:  <input type="text" name="item_title" style=width:600px >
```

Comment:
We could add a `required` tag onto this input element if we wanted; it won't stop deliberate bypassing but it will prevent users accidentally submitting an empty box and save them from a potentially annoying error message

Reply:
Makes sense thanks